custom.css

```diff
@@ -43,6 +43,10 @@ div.callout-note.callout {
    font-size: var(--cdse-default-font-size);
 }
 
+.scrolling {
+   max-height: 300px;
+   overflow-y: auto;
+ }
```

Comment:
Where is this CSS rule used? I don't think we still have to add this here.

Reply:
commented out for the moment instead of removing it because I think was needed for a completely different task that is still in the PR.

Reply:
removed from the css